StantUiIosLibraryDemo/StantUiIosLibraryDemoTests/ConstructionTableViewWithCollapsedSearchBar/ConstructionTableViewWithCollapsedSearchBarTests.swift

```diff
@@ -0,0 +1,141 @@
+//
+//  ConstructionTableViewWithCollapsedSearchBarTests.swift
+//  StantUiIosLibraryDemoTests
+//
+//  Created by Mac Mini Novo on 01/07/19.
+//  Copyright © 2019 Stant. All rights reserved.
+//
+
+import XCTest
+import StantUiIosLibrary
+@testable import StantUiIosLibraryDemo
+
+class ConstructionTableViewWithCollapsedSearchBarTests: XCTestCase {
```

Comment:
Very nice structure and the implementation of tests. I've learned a lot here 👍

Reply:
Cool! :)